### PR TITLE
fix(files): default emptycontent message

### LINKED ISSUE #39443: [F2V] Empty content area always shows a trashbin
When navigating to an empty favorites or shares view the empty content always shows a trashbin icon and related description:

Screenshot 2023-07-17 at 22 12 15

This should probably match the section selected.

cc @skjnldsv

### PR DESCRIPTION
**Pull request by skjnldsv**
Needs https://github.com/nextcloud/nextcloud-vue/pull/4312

Still not sure about the API change, I think we could go a step further. This is still a draft

Fixes
image

Close https://github.com/nextcloud/server/issues/39443

### CODE REVIEW

**Comment by szaimen** [comment]
Tested and some small remarks:

1. the subline text looks distored on mobile: 
image

File: apps/files_sharing/src/views/shares.ts [hunk 1]
```diff
@@ -101,6 +113,9 @@ export default () => {
 		name: t('files_sharing', 'Deleted shares'),
 		caption: t('files_sharing', 'List of shares that you removed yourself from.'),
 
+		emptyTitle: t('files_sharing', 'No deleted shares'),
+		emptyCaption: t('files_sharing', 'Shares you remove yourself from will show up here'),
```
Comment (szaimen) [inline apps/files_sharing/src/views/shares.ts:117]:
```suggestion
		emptyCaption: t('files_sharing', 'Shares from which you removed yourself from will show up here'),
```

File: apps/files_sharing/src/views/shares.ts [hunk 2]
```diff
@@ -57,7 +60,10 @@ export default () => {
 	Navigation.register({
 		id: sharedWithYouViewId,
 		name: t('files_sharing', 'Shared with you'),
-		caption: t('files_sharing', 'List of files that are shared with you.'),
+		caption: t('files_sharing', 'List of files that are shared with you'),
+
+		emptyTitle: t('files_sharing', 'Nothing shared yet'),
+		emptyCaption: t('files_sharing', 'Files and folders you share will show up here'),
```
Comment (szaimen) [inline apps/files_sharing/src/views/shares.ts:66]:
```suggestion
		emptyTitle: t('files_sharing', 'Nothing shared with you yet'),
		emptyCaption: t('files_sharing', 'Files and folders others share with you will show up here'),
```

File: apps/files_sharing/src/views/shares.ts [hunk 3]
```diff
@@ -73,6 +79,9 @@ export default () => {
 		name: t('files_sharing', 'Shared with others'),
 		caption: t('files_sharing', 'List of files that you shared with others.'),
 
+		emptyTitle: t('files_sharing', 'Nothing shared with you yet'),
+		emptyCaption: t('files_sharing', 'Files and folders others share with you will show up here'),
```
Comment (szaimen) [inline apps/files_sharing/src/views/shares.ts:83]:
```suggestion
		emptyTitle: t('files_sharing', 'Nothing shared yet'),
		emptyCaption: t('files_sharing', 'Files and folders you share will show up here'),
```

File: apps/files_sharing/src/views/shares.ts (hunk 1 above)
Comment (skjnldsv) [reply apps/files_sharing/src/views/shares.ts:117]: Not sure tbh. @Pytal any opinion?

`Shares you have removed yourself from will show up here` maybe?

File: apps/files_sharing/src/views/shares.ts (hunk 3 above)
Comment (skjnldsv) [reply apps/files_sharing/src/views/shares.ts:83]: I'm using the previous messages :shrug:

File: apps/files_sharing/src/views/shares.ts (hunk 2 above)
Comment (skjnldsv) [reply apps/files_sharing/src/views/shares.ts:66]: I'm using the previous messages :shrug:

**Comment by skjnldsv** [comment]
> Tested and some small remarks:
> 
> 1. the subline text looks distored on mobile:
>    [image](https://user-images.githubusercontent.com/42591237/252952695-8c4f9235-3613-4de8-bcb1-baed3568ebc5.png)

Something for the vue components then

File: apps/files_sharing/src/views/shares.ts (hunk 2 above)
Comment (szaimen) [reply apps/files_sharing/src/views/shares.ts:66]: Sorry, yes, I should have added a comment

Comment (skjnldsv) [reply apps/files_sharing/src/views/shares.ts:66]: No worries :hugs:

File: apps/files_sharing/src/views/shares.ts (hunk 1 above)
Comment (Pytal) [reply apps/files_sharing/src/views/shares.ts:117]: Since the action is "Leave this share" then "Shares you have left will show up here" would make sense

Comment (skjnldsv) [reply apps/files_sharing/src/views/shares.ts:117]:
```suggestion
		emptyCaption: t('files_sharing', 'Shares you have left will show up here'),
```

**Review by szaimen — APPROVED** [review]
Tested and works

**Review by susnux — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by szaimen** [comment]
Conflicts

**Comment by susnux** [comment]
Rebased

**Pull request merged**